web/src/components/profile/ProfileNonEssentialQuestions.tsx

```diff
@@ -0,0 +1,39 @@
+import { ProfileNonEssentialQuestion } from "@/components/profile/ProfileNonEssentialQuestion";
```

Comment:
Isn't it an anti-pattern to have 2 things named so similarly? (ProfileNonEssentialQuestion vs ProfileNonEssentialQuestions)

Reply:
Thoughts on ProfileNonEssentialQuestion and ProfileNonEssentialQuestionsSection respectively?